### ISSUE TITLE
`cf delete-quota` help text for `-f` flag is incorrect

### ISSUE THREAD

**Issue by ematpl** [cloudfoundry/cli#715]
The `-f` flag should have a description like `Force deletion without confirmation`, not `Can provision instances of paid service plans`.

```bash
$ cf help delete-quota
NAME:
   delete-quota - Delete a quota

USAGE:
   cf delete-quota QUOTA [-f]

OPTIONS:
   -f       Can provision instances of paid service plans

$ cf -v
cf version 6.14.0+2654a47-2015-11-18
```

**Comment by ematpl** [comment]
Looks like it's this line: https://github.com/cloudfoundry/cli/blob/master/cf/commands/quota/delete_quota.go#L28. Seems to have been corrupted in the [Great Codegangsta Purge](https://www.pivotaltracker.com/story/show/97061558).

**Comment by krishicks** [comment]
Thanks, @ematpl!

**Comment by krishicks** [comment]
This is updated in the edge release and will be in the next stable release.

**Comment by dkoper** [comment]
This fix is included in cf CLI release v6.14.1 published today.
https://github.com/cloudfoundry/cli#downloads
